manage.py

```diff
@@ -122,6 +122,9 @@ def init_perl_functions():
         CREATE OR REPLACE LANGUAGE plperl;
       """)
     )
+    # DO NOT EDIT HERE.
```

Comment:
Maybe consider moving this into app/main/lib/shared_models/audio_model.py (or adjacent file?) and just importing into manage.py to run the installation?  Probably would make easier to update if all the code needing updates is in the same place, and that way the actual algorithm logic is not in the control script

Reply:
manage.py is where all our database functions live.